### ISSUE TITLE
 Parse error: syntax error, unexpected ')' in index.php on line 19

### ISSUE THREAD

**Issue by berrabe** [DirectoryLister/DirectoryLister#324]
Parse error: syntax error, unexpected ')' in index.php on line 19

server with ubuntu 18.04.3, nginx, php7.2-fpm

**Comment by PHLAK** [comment]
Thanks. I see the problem.

```php
$app = (new ContainerBuilder)->addDefinitions(
    __DIR__ . '/app/config/app.php',
    __DIR__ . '/app/definitions.php',
)->build()->call(AppManager::class);
```

PHP versions before 7.3 don't support trailing commas in function parameters. Will fix this ASAP.

**Comment by PHLAK** [comment]
Resolved with 57e8e2f
